src/Microsoft.Extensions.DependencyModel/CompilationLibrary.cs

```diff
@@ -51,8 +51,32 @@ public class CompilationLibrary : Library
 
         public IEnumerable<string> ResolveReferencePaths()
         {
+            return ResolveReferencePaths(DefaultResolver);
+        }
+
+        public IEnumerable<string> ResolveReferencePaths(params ICompilationAssemblyResolver[] additionalResolvers)
```

Comment:
> so having additional resolvers last might not solve everything.

The additional resolvers are called first, then the default resolver.

> I'm a bit concerned by performance of doing this per CompilationLibrary.

There should be no difference in perf between what is happening today and this code. Why the concern now?